packages/apollo-server-core/src/errors.test.ts

```diff
@@ -170,5 +172,28 @@ describe('Errors', () => {
         name: 'ValidationError',
       });
     });
+    it('provides a BadUserInput error', () => {
+      const error = new BadUserInputError(message, {
+        field1: 'property1',
+        field2: 'property2',
+      });
+      verifyError(error, {
+        code: 'BAD_USER_INPUT',
+        errorClass: BadUserInputError,
+        name: 'BadUserInputError',
+      });
+      console.log(`error: ${JSON.stringify(error, null, 2)}`);
```

Comment:
We should remove logs from the code. Ideally it would be caught in the linting step. I guess prettier doesn't do that for us anymore 😢

Reply:
Yes thanks for catching that🙏🙏